README.rst

```diff
@@ -30,7 +30,7 @@ The result is a json with the following keys:
 
 Request e.g.::
 
-  GET /api/account.invoice?schema=number,partner_id.name,partner_id.vat,total_amount&filter=[('state','=','open'),('type','in',('out_invoice','out_refund'))]
```

Comment:
@AdriNicolau  I was too fast to approve and merge... this is incorrect:
The first condition uses the `=` operator, then you can't use an array as a parameter, it was correct when was a string.

Reply:
@ecarreras 
I tried it on postman and the backend accept this kind of requests: 
http://localhost/api/res.users?schema=last_location&filter=[('id','=',[3])]

Reply:
Check my message, I was talking about `('state','=','open')` is correct and `('state','=',['open'])` is incorrect because the operator used is **`=`**